### PR TITLE
#160361281 Create UI templates for Fast food fast

### PR DESCRIPTION
**Pull request by sudhons**
**What does this PR do?**
Implement ui templates required for fast-food-fast

**Description of Task to be completed?**
The following interface templates:
- user should see the login/signup page
- user should see a page to make orders
- user should see a page to view previous orders
- Admin should see orders and set status for orders

**What are the relevant pivotal tracker stories?**
#160361281 #160363116 #160363218 #160363109 #160533638

### CODE REVIEW

File: UI/js/kitchen.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const orderDetail = (e) => {
```
Comment (dogoyaro) [inline UI/js/kitchen.js]: Let's use a descriptive variable name. Change `e` to `event`

File: UI/css/style.css [hunk 2]
```diff
@@ -0,0 +1,638 @@
...
+  color: #fff;
+}
+
+.order-table.head {
+  text-transform: uppercase;
+  background-color: #aaa;
+  padding: 0 .3rem;
+}
+
+#new-orders, #accepted-orders,
```
Comment (dogoyaro) [inline UI/css/style.css]: I am not exactly sure why this aren't all in the same class?

File: UI/js/kitchen.js (hunk 1 above)
Comment (sudhons) [reply UI/js/kitchen.js]: I am planning of doing that when refactoring the codes

File: UI/css/style.css (hunk 2 above)
Comment (sudhons) [reply UI/css/style.css]: That's one of the things I am planning of doing when refactoring the code

Comment (dogoyaro) [reply UI/css/style.css]: Please do that in this pull request before merging.

**Comment by dogoyaro** [comment]
Please merge.

**Pull request closed without merging**